bundle/manifests/nodeobservability.olm.openshift.io_nodeobservabilities.yaml

```diff
@@ -51,6 +51,7 @@ spec:
                 - crio-kubelet
                 type: string
             required:
+            - nodeSelector
```

Comment:
nodeSelector is mandatory?

Reply:
[I asked](https://github.com/openshift/node-observability-operator/pull/83#discussion_r943935746) Ayesha to make it mandatory. I think it makes sense because we don't have a meaningful behavior in case the nodeSelector is not specified (only possible with optional).
What happens now (with this PR) if the node selector is not specified: agent's daemon gets scheduled on all the worker nodes as the nodeSelector is not there. MC and MCP are created but no nodes get labeled with `nodeObservability` role, so no CRIO pprof configuration is applied. Further `NodeObservabilityRun`s won't work as the pprof unix socket is not setup.